src/Stryker.Core/Stryker.Core/Reporters/HtmlReporter/HtmlReporter.cs

```diff
@@ -38,22 +37,30 @@ public void OnAllMutantsTested(IReadOnlyProjectComponent reportComponent)
 
             WriteHtmlReport(reportPath, mutationReport.ToJsonHtmlSafe());
 
-            // to make path clickable it should always start with: file:///
-            var reportUri = reportPath.Replace("\\", "/");
-            reportUri = reportUri.StartsWith("/") ? reportUri : "/" + reportUri;
-            reportUri = "file://" + reportUri;
+            var reportUri = "file://" + reportPath.Replace("\\", "/");
 
             if (_options.ReportTypeToOpen == Options.Inputs.ReportType.Html)
             {
                 _processWrapper.Open(reportUri);
-            } else
+            }
+            else
+            {
+                _console.MarkupLine("[Cyan]Hint: by passing \"--open-report or -o\" the report will open automatically once Stryker is done.[/]");
+            }
+
+            _console.WriteLine();
+            _console.MarkupLine("[Green]Your html report has been generated at:[/]");
+
+            if (_console.Profile.Capabilities.Links)
+            {
+                _console.MarkupLine($"[Green][link={reportUri}]{reportPath}[/][/]");
+            }
+            else
             {
-                _consoleWriter.Write(Output.Cyan("Hint: by passing \"--open-report or -o\" the report will open automatically once Stryker is done."));
+                _console.MarkupLine($"[Green]{reportPath}[/]");
```

Comment:
I think we should print the report uri and not the report path if ansi links are not supported, because some terminals have built-in uri detection but not ansi link support. For a user with a terminal that supports neither nothing changes as browsers support full file uri's in general. It would provide the largest cover.